src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CatalogueSolutionsController.cs

```diff
@@ -155,27 +158,50 @@ public ActionResult<CreateOrderItemModel> GetOrderItem(string orderId, string or
         }
 
         [HttpPut]
-        [Route("{orderItemId}")]
+        [Route("{orderItemId:int}")]
```

Comment:
Did you try the route without the :int? Did it still work. I just dont think you need it as you're specifiying the parameter underneith (but I could be wrong)

Reply:
According to this [website](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-3.1#route-constraint-reference), route constraints should not be used for input validation. I will remove this constraint.